stix_shifter_modules/qradar/stix_translation/json/aql_events_fields.json

```diff
@@ -25,7 +25,6 @@
     "identityhostname",
     "eventcount",
     "PROTOCOLNAME(protocolid) as protocol",
-    "BASE64(payload) as base64_payload",
     "UTF8(payload) as utf8_payload",
```

Comment:
This could be renamed as `eventpayload` to differentiate it from the flow payloads in the to-stix map.

Reply:
right. made that change